tests/unit/ts_utils_spec.lua

```diff
@@ -0,0 +1,43 @@
+local tsutils = require "nvim-treesitter.ts_utils"
+
+describe("is_in_node_range", function()
+
+  function test_is_in_node_range(line, col, expected)
+    local node = {
+      range = function ()
+        return unpack({0, 3, 2, 5})
+      end
+    }
+    return tsutils.is_in_node_range(node, line, col)
+  end
+
+  it("false before node start", function ()
```

Comment:
Maybe add a `returns` here, this is just wording but it really sounds better afterwards.

Reply:
Yeah, should i add it for each `it`, or can i do this just for `describe`?

Reply:
I would add it to each `it`, so that it reads as `it('returns something',...)`.